examples/library/test_manycore_alignment/main.c

```diff
@@ -164,7 +163,7 @@ int test_manycore_alignment() {
                                                  dram_coord_y,
                                                  BASE_ADDR + 1);
                         write_data = rand();
-                        err = hb_mc_manycore_write32(mc, &npa, &write_data);
+                        err = hb_mc_manycore_write32(mc, &npa, write_data);
```

Comment:
Is this just an outdated test? What went wrong here?

Reply:
The `hb_mc_manycore_write*` functions call the `hb_mc_manycore_write` functions by passing the address of the third argument (i,e write_data here). Passing the third argument as an address, would make the call to write a pointer to a pointer. Is that the intention? If it is, I am wrong and I will revert it.

Reply:
No, you're probably right. Just confused why this wasn't caught before. Does the test pass?

Reply:
Yes.

Reply:
huh...

Reply:
It passes with the changes I have. I can check it without the changes xP

Reply:
Both versions of the test (with the changes I proposed and without them) work. So, I am not sure what is the course of action of here. Keep it the same?

Reply:
Oh an always passing test, that’s a classic. 

Course of action is revert the change and file an issue

Reply:
Opened #713 for this. 

Also found that the test_pod_iteration library test fails due to an `Invalid Input` error raised by the `hb_mc_platform_init` function. Opened #712 for that